core/controllers/contributor_dashboard.py

```diff
@@ -996,13 +996,10 @@ def get(
         self, username: str, suggestion_type: str
     ) -> None:
         """Handles GET requests."""
-        language = None
         assert self.normalized_request is not None
         from_date = self.normalized_request['from_date']
         to_date = self.normalized_request['to_date']
-
-        if suggestion_type == feconf.SUGGESTION_TYPE_TRANSLATE_CONTENT:
-            language = self.normalized_request['language']
+        language = self.normalized_request.get('language')
```

Comment:
```suggestion
        language = self.normalized_request['language']
```

Reply:
I think we will have to use the `self.normalized_request.get('language')` because when generating the question certificate, we do not include language param in the request and it throws `KeyError: 'language'` if we use `self.normalized_request['language']`

We can get the the default value which is set as `None` if we use `self.normalized_request.get('language')` when language is not included in the request.

Reply:
Ah, right, sorry.